docker/run

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+TGT_DIR=$(pwd)  # the target folder where we run cow from
+
+export USER_ID=$(id -u)
+
+cd $(dirname $0) && \
```

Comment:
If you create a symlink the the run file and try calling cow through the symlink, `cd $(dirname $0)` will return the path of the symlink, which means you won't get all the related docker file.

This seams to work for me. Got it from SO https://stackoverflow.com/questions/59895/get-the-source-directory-of-a-bash-script-from-within-the-script-itself

```suggestion
DIR=`dirname "$(readlink -f "$0")"`
cd $DIR && \
```

Reply:
yep, fair enough... silly mistake :D